sdk/identity/test-resources-pre.ps1

```diff
@@ -23,9 +24,11 @@ $sshKey = Get-Content $PSScriptRoot/sshKey.pub
 
 $templateFileParameters['sshPubKey'] = $sshKey
 
-# Get the max version that is not preview and then get the name of the patch version with the max value
+az cloud set --name $Environment
 az login --service-principal -u $TestApplicationId --tenant $TenantId --allow-no-subscriptions --federated-token $env:ARM_OIDC_TOKEN
-$versions = az aks get-versions -l westus -o json | ConvertFrom-Json
+# Get the max version that is not preview and then get the name of the patch version with the max value
+$region = if ($Environment -eq 'AzureUSGovernment') { 'usgovvirginia' } elseif ($Environment -eq 'AzureChinaCloud') { 'chinanorth3' } else { 'westus' }
```

Comment:
Seems like something else we might want to export at some point.

Reply:
Ideally we pass `Location` as a parameter down to the pre/post scripts the same way we do Environment. I checked but don't think I saw a place where we are doing it?